package.json

```diff
@@ -28,10 +28,6 @@
     "@octokit/plugin-throttling": "^4.0.0",
     "quick-format-unescaped": "^4.0.1"
   },
-  "devDependencies": {
-    "semantic-release": "^19.0.0",
-    "semantic-release-plugin-update-version-in-files": "^1.1.0"
```

Comment:
we will need to install `semantic-release-plugin-update-version-in-files` before running `npx semantic-release` in `release.yml`

Reply:
I doubted if that would be necessary. So I assume the `semantic-release` plugins under `@semantic-release` scope are accessible out of the box, is that correct?